src/components/ProjectList/listOfProjects.js

```diff
@@ -661,6 +661,12 @@ const projectList = [
     projectLink: 'https://github.com/mattermost/mattermost-server/contribute',
     description: 'Open source Slack-alternative for DevOps teams',
     tags: ['Go', 'Javascript', 'React', 'React Native'],
+  }, {
+    name: 'Markdown Dungeon',
+    imageSrc: 'https://avatars3.githubusercontent.com/u/67384272?v=4',
+    projectLink: 'https://github.com/MakeContributions/markdown-dungeon',
```

Comment:
Would be better if you point to the issues page or the /contribute page of the project instead, we follow those links in the `projectLink` tag.

Reply:
Done, check it oyt